src/FsCheck/Runner.fs

```diff
@@ -120,6 +122,42 @@ module Runner =
                     yield EndShrink result
               yield! test newSize resize rnd1 gen
         }
+        
+#if PCL // sorry windows phone no parallelism for you
+    let private parallelTest limit = test
+#else
+    let internal parallelTest limit initSize resize rnd0 gen =
+        (initSize, rnd0)
+        |> Seq.unfold (fun (initSize, rnd) ->
+            let rnd1, rnd2 = Random.split rnd
```

Comment:
> If I understand you properly then you are about to rewrite sequential test like this:

Yes, that's what I had in mind.

>My concern is to keep results of parallel execution consistent with results of sequential algorithm. Otherwise user will have hard times if he first found falsifiable seed in par and then switching to seq with same seed for debugging.

That is a valid concern, I think largely addressed by #310 with the ability to run a single test case with a given seed and size (i.e. one case, trivially sequential).

The advantage of using `split` for parallel execution is that you could parallelize the whole thing, whereas now you basically have a purely sequential part first to generate the random values and size. Perhaps this is unlikely to make a big difference in practice - generating random values should be pretty fast. I also don't know if it would work well with the parallelism API. But I thought I'd mention it in case you are interested to explore the option or the benchmarks don't come out looking totally awesome ;)

For this PR, I'd be  totally fine with keeping things as is (i.e. using `split`) and we can look at using `next` instead later. If we keep the seeds as pre-generated in the parallel run, switching `split` to `next` should be trivial to do later anyway.